README.md

```diff
@@ -328,7 +328,7 @@ See [Advanced configuration](#advanced-configuration) for details and more confi
 
     ~~~ fish
     set -Ux PYENV_ROOT $HOME/.pyenv
-    set -U fish_user_paths $PYENV_ROOT/bin $fish_user_paths
+    fish_add_path $PYENV_ROOT/bin
```

Comment:
`fish_add_path` is new in Fish 3.2.0. There are still non-EOL environments that provide earlier versions -- e.g. [Focal](https://packages.ubuntu.com/focal/fish).

So atm this can probably only be offered as an option "if you have Fish 3.2.0+".

Reply:
I've adjusted the instructions to offer this choice. PTAL